src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/catalog.cljs

```diff
@@ -154,8 +163,41 @@
                                                :disabled disabled?
                                                :placeholder inputHint
                                                :value (get (:attributes @state) property-kwd)
+
                                                :onChange update-property}))]))
-          questions)]))})
+          questions)]))
+   :component-did-mount
+   (fn [{:keys [props state refs]}]
+     (let [{:keys [library-schema questions]} props]
+       (doseq [{:keys [property]} questions]
+         (let [property-kwd (keyword property)
+               {:keys [typeahead relatedID]} (get-in library-schema [:properties property-kwd])
+               options  (js/Bloodhound. (clj->js
+                                          {:datumTokenizer js/Bloodhound.tokenizers.whitespace
+                                           :queryTokenizer js/Bloodhound.tokenizers.whitespace
+                                           :remote (clj->js {:url (str (config/api-url-root) "/duos/autocomplete/%QUERY")
+                                                             :wildcard "%QUERY"})}))]
+             (if (= typeahead "ontology")
+               (do
+                 (.typeahead (js/$ (@refs property))
+                             (clj->js {:highlight true
+                                       :hint true
+                                       :minLength 1})
+                             (clj->js
+                               {:source options
+                                :display (fn [result]
+                                           (aget result "label"))
+                                :templates (clj->js
+                                             {:empty "<div> unable to find any matches to the current query </div>"
+                                              :suggestion
+                                              (fn [result]
+                                                (str "<div> <div style='font-weight: bold; line-height: 1.5em;'>" (aget result "label")
```

Comment:
by making the label bold, the `highlight` feature of the typeahead library doesn't work for label - because it's trying to bold something that's already bold. Highlighting is cool - maybe this needs some CSS tweaks?